src/node_contextify.cc

```diff
@@ -236,7 +236,13 @@ class ContextifyContext {
     Local<String> script_source(args[0]->ToString(args.GetIsolate()));
     if (script_source.IsEmpty())
       return;  // Exception pending.
-    Context::Scope context_scope(Debug::GetDebugContext());
+
+    Environment* env = Environment::GetCurrent(args.GetIsolate());
```

Comment:
Actually, I meant to say that it'll be executed in 99.999% of cases anyway, not 0%.